libs/build_mapl.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -eux
+
+name="mapl"
+repo=${1:-${STACK_mapl_repo:-"GEOS-ESM"}}
```

Comment:
Here as well.

Reply:
Done in 81e539f4a6443c4253b9e4825eeeebcd28bda884